### ISSUE TITLE
Unable to navigate dropdown component using arrow keys and For non Mobile applications ensure that, with the border or outline, the keyboard focus indicator is visible to users

### ISSUE THREAD

**Issue by reeteshsingh93** [carbon-design-system/carbon#2813]
## Environment

> Operating system

Mac os mojave

> Browser
Chrome

> Automated testing tool and ruleset

> Assistive technology used to verify

## Detailed description

> What version of the Carbon Design System are you using?
  "carbon-components": "9.57.0",
    "carbon-components-react": "6.54.1",
    "carbon-icons": "7.0.7",

> What did you expect to happen?

A visual focus indicator might be a vertical bar (a typical cursor) or a visible border surrounding a user interface control.

Try these techniques:
Use interface components that are highlighted by the user agent when they receive focus
Use CSS to change the presentation of a user interface component when it receives focus
Use a script to change the background color or border of the element when it receives focus
IBM Accessibility Checklist Reference
IBM 2.4.7 Focus Visible

> What happened instead?
 Dropdown is not getting highlighted

> What WCAG 2.1 checkpoint does the issue violate?
IBM 2.4.7 Focus Visible

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

> Please create a reduced test case in CodeSandbox
> https://codesandbox.io/s/github/IBM/carbon-components-react/tree/master/examples/codesandbox

## Additional information

- Screenshots or code

![image](https://user-images.githubusercontent.com/46310374/58228582-ce154e00-7d4c-11e9-95fe-109fd4d2b178.png)

- Notes

**Comment by dakahn** [comment]
Hello and thanks for submitting.

Please provide a reduced test case at the link in the template. Looking at https://v9.carbondesignsystem.com/components/dropdown/code

We have a visual indicator and arrowkey navigation functionality.
